examples/ucp_client_server.c

```diff
@@ -268,20 +269,24 @@ static int send_recv_stream(ucp_worker_h ucp_worker, ucp_ep_h ep, int is_server,
                             int current_iter)
 {
     char recv_message[TEST_STRING_LEN]= "";
+    ucp_request_param_t param;
     test_req_t *request;
     size_t length;
 
+    param.op_attr_mask = UCP_OP_ATTR_FIELD_CALLBACK | UCP_OP_ATTR_FIELD_DATATYPE;
```

Comment:
pls change the example to use user_data to point to &complete flag, instead of extending request size

Reply:
done... hello_world example update too?